crates/trigger/src/runtime_config/llm.rs

```diff
@@ -1,13 +1,39 @@
+use spin_llm_local::LocalLlmEngine;
+use spin_llm_remote_http::RemoteHttpLlmEngine;
+use url::Url;
+
 pub(crate) async fn build_component(
     runtime_config: &crate::RuntimeConfig,
     use_gpu: bool,
 ) -> spin_llm::LlmComponent {
-    spin_llm::LlmComponent::new(
-        runtime_config
-            .state_dir()
-            .unwrap_or_default()
-            .join("ai-models"),
-        use_gpu,
-    )
-    .await
+    match runtime_config.llm_compute() {
+        LlmComputeOpts::Spin => {
+            let path = runtime_config
+                .state_dir()
+                .unwrap_or_default()
+                .join("ai-models");
+            let mut engine = LocalLlmEngine::new(path, use_gpu);
+            engine.init().await;
+            spin_llm::LlmComponent::new(move || Box::new(engine.clone()))
+        }
+        LlmComputeOpts::RemoteHttp(config) => {
+            println!("Using remote compute for LLMs");
```

Comment:
Could we remove this line please?  The Spin prologue is already long enough that nobody will ever read it... prologue items really only have value when we are telling people where to find a thing that they might want to look at outside of Spin (like the logs or the database), and this doesn't meet that criterion.  You could put it as an `info!` or `debug!` trace if you feel it's useful?

Reply:
done